commerce-payment-method-authorize-net/src/main/java/com/liferay/commerce/payment/method/authorize/net/internal/AuthorizeNetCommercePaymentMethod.java

```diff
@@ -96,6 +96,12 @@ public CommercePaymentResult completePayment(
 			CommercePaymentRequest commercePaymentRequest)
 		throws Exception {
 
+		if (!(commercePaymentRequest instanceof
+				AuthorizeNetCommercePaymentRequest)) {
+
+			throw new ClassCastException();
```

Comment:
Does this buy us anything? Wouldn't the next statement down throw a `ClassCastException` without this diff?

EDIT: same for all the diffs like this below.

Reply:
Yeah I'd imagine the logic before was fine. This is just the change that the javadoc requested